Python/Prerequisites/deploy_test_targets.cmd

```diff
@@ -38,4 +35,21 @@ if not exist "%TARGET%" mkdir "%TARGET%"
 copy /Y "*.targets" "%TARGET%"
 popd
 
+set TARGET=%VSINSTALLDIR%\MSBuild\Microsoft\VC\v160\Platforms
```

Comment:
Sorry forgot I was working on testing this at the same time. This isn't shipping code so should be fine to submit too.